CSSampleApp/src/main/java/com/example/androidsampleapp/ComposeSimpleActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.androidsampleapp
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.contentsquare.android.Contentsquare
+import com.contentsquare.android.compose.analytics.TriggeredOnResume
+
+class ComposeSimpleActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            MaterialTheme {
+                // Display simple compose screen
+                SimpleScreen()
+            }
+        }
+    }
+}
+
+@Composable
+fun SimpleScreen() {
```

Comment:
Where did you get the idea for what to display because this does look VERY simple.

Reply:
Yes, it's simple :) It was specified like this in the task: https://contentsquare.atlassian.net/browse/MOBILE-6754

Reply:
The screen should look like the existing ones. It's not about the contents, it's about sending the screenview.